apps/server/src/modules/learnroom/service/course-sync.service.spec.ts

```diff
@@ -350,20 +397,25 @@ describe(CourseSyncService.name, () => {
 			});
 		});
 
-		describe('when the last teacher gets removed from a synced group', () => {
+		describe('when the teachers are not synced from group (partial sync)', () => {
```

Comment:
the test: 'when the last teacher gets removed from a synced group' must not be removed 😅